tests/acceptance/common.py

```diff
@@ -268,51 +268,53 @@ def common_boot_from_internal(conn):
 
 def latest_build_artifact(builddir, extension, sdimg_location=None):
     global configuration
-    
+
     if configuration.get("conversion"):
-        output = subprocess.check_output(["sh", "-c", "ls -t %s/%s/*%s | grep -v data*%s| head -n 1" % (builddir, sdimg_location, extension, extension)])
+        output = subprocess.check_output(["sh", "-c", "ls -t %s/*%s | grep -v data*%s| head -n 1" % (builddir, extension, extension)])
     else:
         output = subprocess.check_output(["sh", "-c", "ls -t %s/tmp*/deploy/images/*/*%s | grep -v data*%s| head -n 1" % (builddir, extension, extension)])
     output = output.decode().rstrip('\r\n')
-    print("Found latest image of type '%s' to be: %s" % (extension, output))
-    return output
+    path_to_image = os.path.abspath(output)
```

Comment:
Just a slight detail, but in this case. maybe we can remove above `rstrip('\r\n')` call?

Reply:
I have reworked this change.